app/poros/animal_detail.rb

```diff
@@ -15,14 +15,14 @@ def initialize (animal_data, photo_data)
     @name = animal_data[:name]
     @photo_url = photo_data[:photos].first[:url]
     @prey = (animal_data.dig(:characteristics, :main_prey) || animal_data.dig(:characteristics, :prey))
-    @predators = animal_data.dig(:characteristics, :predators)
-    @habitat = animal_data.dig(:characteristics, :habitat)
+    @predators = (animal_data.dig(:characteristics, :predators) || "Human")
```

Comment:
This fixes our null issue, would be fun to figure out how to dry this out and maybe fun sayings if we can't.